crates/solidity/testing/utils/src/cst_snapshots/mod.rs

```diff
@@ -114,17 +110,29 @@ fn write_node<W: Write>(
 ) -> Result<()> {
     let range_string = format!(
         "{range:?}",
-        range = node.range.start.byte..node.range.end.byte
+        range = node.range.start.utf8..node.range.end.utf8
     );
 
     let (node_value, node_comment) = if node.range.is_empty() {
         (" []".to_owned(), range_string)
     } else {
         let preview = node.render_preview(source, &node.range)?;
         if node.children.is_empty() {
-            (format!(" {preview}"), range_string)
+            (
+                format!(" {preview}"),
+                format!(
+                    "{range:?}",
+                    range = node.range.start.utf8..node.range.end.utf8
```

Comment:
super nit: these two are perfect candidates for the "range_utf8" util added earlier.

Reply:
done